Makefile

```diff
@@ -43,8 +44,10 @@ docker-build:
 
 .PHONY: docker-push
 docker-push:
+ifdef IMG_REPO
```

Comment:
Maybe I'm wrong, but won't it always be defined (at least containing just "/" )?

Reply:
aaah.. sorry, my fault